src/Core/Binding/BoundServerProject.cs

```diff
@@ -43,4 +45,35 @@ public BoundServerProject(string localBindingKey, string serverProjectKey, Serve
         ServerConnection = serverConnection ?? throw new ArgumentNullException(nameof(serverConnection));
         LocalBindingKey = localBindingKey;
     }
+
+    public static BoundServerProject FromBoundSonarQubeProject(BoundSonarQubeProject boundProject)
```

Comment:
There are no separate tests for these methods as they're temporary. Once the ticket for solution binding repo is done, there will be tests for model conversion